src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -41,6 +46,12 @@ const MSG = defineMessages({
   },
 });
 
+enum TabName {
+  SuggestionsTab = 'suggestions',
+  TasksTab = 'tasks',
+  TransactionsTab = 'transactions',
```

Comment:
@chmanie These enum keys are a bit wordy (i.e. `SuggestionsTab` vs `Suggestions`) as I was experiencing a false-positive from the eslint `no-shadow` rule. Not really sure why that rule is applied to keys... but it is.